general-configuration-guide/basic-setup.md

```diff
@@ -74,101 +73,63 @@ tinymce.init({
-superscript | | | |
-
-![{{site.productname}} Default Toolbar & Menubar State]({{ site.baseurl }}/images/tinymce-default-state.png)
-
-*Default {{site.productname}} toolbar & menubar state*
-
-The `toolbar` option displays, orders, and groups the toolbar buttons.
-
-Use a space-separated list to specify the buttons that appear in {{site.productname}}'s toolbar. Use `"|"` pipe characters between the groups of buttons to create groups within this list.
-
-##### Example
-
 ```js
 tinymce.init({
   selector: 'textarea',  // change this value according to the HTML
-  toolbar: 'undo redo | styleselect | bold italic | link image',
+  toolbar: 'undo redo | styleselect | bold italic | alignleft aligncenter alignright alignjustify | outdent indent',
 });
 ```
 
-Set the boolean value to `false` to disable the toolbar entirely. This example disables the toolbar.
-
-```js
+<script>
 tinymce.init({
-  selector: 'textarea',  // change this value according to the HTML
-  toolbar: false
+  selector: 'textarea.default',
 });
-```
+</script>
+<textarea class="default">Hello, World!</textarea><br/>
 
-The example below specifies multiple toolbars by providing an `array` of space-separated strings.
+The `toolbar` option defines the presence, order, and grouping of toolbar buttons.
 
-##### Example
+Use a space-separated list to specify the toolbar buttons for {{site.productname}}. Create toolbar groups by using the "`|`" pipe character between buttons.
 
-```js
-tinymce.init({
-  selector: 'textarea',  // change this value according to the HTML
-  toolbar: [
-    'undo redo | styleselect | bold italic | link image',
-    'alignleft aligncenter alignright'
-  ]
-});
-```
-> Note: Adding multiple `toolbar` options suffixed with a number i.e. "toolbar1", "toolbar2", "toolbar3" is allowed.
+#### Additional information
 
+* For additional toolbar options, including disabling the toolbar and adding multiple toolbars, see: [User interface options - toolbar]({{site.baseurl}}/configure/editor-appearance/#toolbar).
+* For a list of all available toolbar items, see: [Editor control identifiers - Toolbar controls](https://www.tiny.cloud/docs/advanced/editor-control-identifiers/#toolbarcontrols).
 
-### A quick look at the menu and menubar controls
+## Menu and Menu bar configuration
 
-Just as there is a `toolbar` option, there is also a `menu` option. There  are two menu related options: `menu` and `menubar`. Let's take a quick look at the default menu items. Some of these settings are used in the examples throughout this guide and all of them within the plugins and customization documentation.
+There are two menu options: `menubar` and `menu`.
+`menubar` is used to define the presence and order of _menus_, such as **File**, **Edit**, and **View**.
+`menu` is used to define the presence and order of _menu items_, such as **New Document**, **Select all**, and **Source code**.
 
-#### Default menu controls
+### Menu and Menubar Examples
```

Comment:
This is inconsistent with the above heading?

Reply:
updated to: Menu and Menu bar configuration examples